File: src/main/java/com/example/ranyoungonboarding/repository/SurveyRepository.java
package com.example.ranyoungonboarding.repository;


import com.example.ranyoungonboarding.domain.Survey;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SurveyRepository extends JpaRepository<Survey, Long> {

    //기본 CRUD는 JpaRepository에서 제공
}

File: project/minjong-onboarding/src/main/kotlin/com/innercircle/survey/repository/SurveyRepository.kt
// src/main/kotlin/com/innercircle/survey/repository/SurveyRepository.kt
package com.innercircle.survey.repository

import com.innercircle.survey.domain.Survey
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface SurveyRepository : JpaRepository<Survey, Long> {
    // 기본 CRUD 메서드는 JpaRepository에서 자동 제공됨
}